public/app/features/dashboard/all.js

```diff
@@ -16,4 +16,5 @@ define([
   './graphiteImportCtrl',
   './dynamicDashboardSrv',
   './importCtrl',
+  './impressionStore',
```

Comment:
please never use uppercase letters in file names, use underscore, impression_store.js